test/integrations/destinations/klaviyo/processor/identifyTestData.ts

```diff
@@ -425,80 +428,6 @@ export const identifyData = [
   {
     id: 'klaviyo-identify-test-6',
     name: 'klaviyo',
-    description: 'Identify call without user custom Properties',
```

Comment:
@Gauravudia what is the need to delete the test case? It will add the newly mapped properties only right? Or is the test case itself is repeating

Reply:
With new changes, and current payload we have, identify call will have the custom properties. So, this testcase is not relevant anymore.

Reply:
These mapping changes right? Is custom property something else? is it introduced in some earlier release?

Reply:
updated the testcase